examples/python/signed_distance_fields/signed_distance_fields/__main__.py

```diff
@@ -117,7 +117,14 @@ def log_sampled_sdf(points: npt.NDArray[np.float32], sdf: npt.NDArray[np.float32
 
 def log_volumetric_sdf(voxvol: npt.NDArray[np.float32]) -> None:
     names = ["width", "height", "depth"]
-    rr.log("tensor", rr.Tensor(voxvol, dim_names=names))
+    # Use a symmetric value range, so that the `cyantoyellow` colormap centers around zero.
+    # Either positive or negative range might be quite small, so don't exceed 1.5x of either.
+    min_val = cast(float, np.min(voxvol))
+    max_val = cast(float, np.max(voxvol))
+    negative_range = max(0.0, abs(min_val))
+    positive_range = max(0.0, abs(max_val))
```

Comment:
an absolute value cannot be less than 0.0

```suggestion
    negative_range = abs(min_val)
    positive_range = abs(max_val)
```

Reply:
oof, must have been late or something